docs/user/rollout-policies.md

```diff
@@ -0,0 +1,107 @@
+# Rollout Device Selection and Rollout Disruption Budget
+
+## Overview
+
+When performing a rollout using `flightctl`, it is crucial to manage which devices participate in the rollout and how much disruption is acceptable. This document outlines the device selection process and the concept of the rollout disruption budget to ensure controlled and predictable rollouts.
+
+---
+
+## Rollout Device Selection
+
+### **Device Targeting**
+
+A rollout applies only to devices that belong to a fleet. Each device can belong to only a single fleet. Since rollout definitions are done at the fleet level, the selection process determines which devices within a fleet will participate in the rollout based on label criteria.
+
+- **Labels**: Devices with specific metadata labels can be targeted for rollouts.
+- **Fleet Membership**: Rollouts apply only to devices within the specified fleet.
+
+### **Device Selection Strategy**
+
+Currently, Flightctl supports only the **BatchSequence** strategy for device selection. This strategy defines a stepwise rollout process where devices are added in batches based on specific criteria.
+
+Batches are executed sequentially. After each batch completes, execution proceeds to the next batch only if the success rate of the previous batch meets or exceeds the configured **success threshold**. The success rate is determined as:
+
+```text
+# of successful rollouts in the batch / # of devices in the batch >= success threshold
+```
+
+A **successful rollout** is when a device completes the rollout process successfully.
```

Comment:
This is obvious.  It would be more helpful to specify the criteria.  For example, the status field(s) that you look at.

Reply:
Will remove